compare_rgbw.py

```
@@ -0,0 +1,360 @@
+
+def fmt(rgbw):
+    return "(%3d,%3d,%3d,%3d)" % rgbw
+
+
+def delta(a, b):
+    return tuple(int(b[i]) - int(a[i]) for i in range(4))
+
+
+def main():
+    print(
+        "Profile: %d×%d×%d × %d-LED primaries from reference; source = Rec709/D65 (linear)"
```

Ruff (0.15.15)

[warning] String contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF001)